test/integration/quota/quota_test.go

```diff
@@ -355,10 +357,10 @@ func TestQuotaLimitedResourceDenial(t *testing.T) {
 	if err != nil {
 		t.Fatalf("unexpected err: %v", err)
 	}
-	go resourceQuotaController.Run(ctx, 2)
+	go resourceQuotaController.Run(ctx, 1)
 
 	// Periodically the quota controller to detect new resource types
-	go resourceQuotaController.Sync(discoveryFunc, 30*time.Second, ctx.Done())
+	go resourceQuotaController.Sync(discoveryFunc, controllerSyncPeriod, ctx.Done())
```

Comment:
same question here... we're not adding/removing defined types (e.g. CRDs) in this test, so why do we care how often this sync happens?

Reply:
sync period has been reverted to 30s